### DISCUSSION TITLE
500 error when attempting to view project files.

### DISCUSSION THREAD

**Discussion by ukane-philemon** [community/community#48825] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug
<img width="1621" alt="Screenshot 2023-02-28 at 7 34 45 PM" src="https://user-images.githubusercontent.com/57448127/221982931-5dd4d24f-21c5-4a46-b227-49de8f850b2d.png">

### Body

I recently notice am unable to view repository files(for any project) on the browser without getting a 500 server error.

EDIT: Every other thing looks fine, except viewing files of any project under `/tree` or `/blob` paths returns an error page.

**Comment by vishnubansal001** [comment]
Hi @ukane-philemon Bro,
I have opened your id and I am able to see any of the repository's code and all its files there can be a network or some other cause of this problem please try again

**Reply by ukane-philemon** [reply]
Darn, tried another broswer and the issue seems to go away. However, it mysteriously remains a issue on my chrome browser.

**Reply by vishnubansal001** [reply]
Ya @ukane-philemon bro,
I am also using another browser which is brave not chrome i don't now what's error in chrome but on my system chrome is working fine 
Hope it is helpful. Please Mark my answer as Answer
![Screenshot (81)](https://user-images.githubusercontent.com/101445306/221994091-2d94c682-6fb8-4756-b93d-3c59f0319b6a.png)

you can see i m not getting this error

**Reply by ukane-philemon** [reply]
I flushed my cookies and that fixed the issue. Thanks for your comment.

**Comment by ukane-philemon** [comment, marked as answer]
For the sake of others and for @salemalamri00 to stop spamming this thread and my mail, the answer is:

**Flush your cookies and browser cache.**
